packages/discovery-provider/src/api/v1/users.py

```diff
@@ -1572,8 +1573,8 @@ def _get(self, id):
         offset = get_default_max(args.get("offset"), 0)
         current_user_id = get_current_user_id(args)
         args = {
-            "follower_user_id": decoded_id,
-            "current_user_id": current_user_id,
+            "followee_user_id": decoded_id,
+            "follower_user_id": current_user_id,
```

Comment:
I think the follower/followee pattern is confusing. But I see we're sticking to the `id` / `userId` pattern in the SDK methods/API so it's not a huge deal.